### PR TITLE
MoreDrillsWithRSpec completed by skirby

### PR DESCRIPTION
**Pull request by SDKirbside**
@jaybobo @sampriddy @sjreich

### CODE REVIEW

File: lib/fizzbuzz.rb [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
+    if input % 3 == 0
+      output << "Fizz"
+    end
 
-  end
+    if input % 5 == 0
+      output << "Buzz"
+    end
 
+    return output != "" ? output : input
```
Comment (sjreich) [inline lib/fizzbuzz.rb]: #empty? might be useful here.  Also, the `return` statement is unnecessary.

**Comment by sjreich** [comment]
:+1:  on fizzbuzz.

File: lib/deaf_grandma.rb [hunk 2]
```diff
@@ -14,18 +14,21 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
```
Comment (sjreich) [inline lib/deaf_grandma.rb]: why `p` and not `puts` here?

**Comment by sjreich** [comment]
and :+1: on deaf grandma.  Very nice.

File: lib/deaf_grandma.rb (hunk 2 above)
Comment (jaybobo) [inline lib/deaf_grandma.rb]: @sjreich `p` gives you the same value as `.inspect` but without `puts`'s newline.

File: spec/deaf_grandma_spec.rb [hunk 3]
```diff
@@ -9,10 +9,12 @@
...
   it "says 'NOT SINCE 1964!' when we yell" do
-    #implement your test here
+    expect(script.speak("HI GRANDMA")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
-    #implement your test here
+    script.speak("BYE")
+    script.speak("BYE")
+    expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
```
Comment (degoeym) [inline spec/deaf_grandma_spec.rb]: Can you figure out how to tighten up this test so it's done in fewer lines?

File: lib/deaf_grandma.rb (hunk 2 above)
Comment (sjreich) [inline lib/deaf_grandma.rb]: I know.  I just don't know why that's desirable in this case. :smile_cat:

**Pull request closed without merging**